src/game.js

```diff
@@ -0,0 +1,112 @@
+import Tile from './tiles';
+
+class Game {
+  constructor(size) {
+    const tilesWin = [];
+    for (let i = 0; i < size * size - 1; i++) {
+      tilesWin[i] = i + 1;
+    }
+    tilesWin[size * size - 1] = '';
+
+    this.tilesWin = tilesWin;
+    this.tilesWin1514 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 15, 14, ''];
+    this.length = size * size;
+
+    this.container = document.createElement('DIV');
+    this.renderTiles = this.renderTiles.bind(this);
```

Comment:
I would suggest to move all bind() to a separate method in order our constructor looks clean